packages/suite/src/hooks/suite/useDiscoveryButtonVisible.ts

```diff
@@ -0,0 +1,18 @@
+import { useEffect, useState } from 'react';
+import { useDiscovery, useSelector } from 'src/hooks/suite';
+import { selectDiscoveryForDevice } from '@suite-common/wallet-core';
+
+export const useDiscoveryButtonVisible = () => {
+    const [isDiscoveryButtonVisible, setIsDiscoveryButtonVisible] = useState(false);
+
+    const discovery = useSelector(selectDiscoveryForDevice);
+    const { isDiscoveryRunning } = useDiscovery();
+    useEffect(() => {
+        if (discovery && !isDiscoveryRunning && discovery.loaded < discovery.total) {
```

Comment:
I'm not sure about this condition: `discovery.loaded < discovery.total`

Are there some cases when it's not going to work? I tested it and it worked fine, but I didn't know how exactly know that discovery needs to be done.

Reply:
Seems to me like that should be the condition! Basically, we need to tell some UI `Do smth when there is a need to re-discover due to a mismatch between discovered accounts and what should be discovered`.

 @marekrjpolak, could you confirm, since you are the Discovery Master? 👨‍💻